### PR TITLE
[Math] Add solution to AddDigits

### PR DESCRIPTION
**Pull request by Quaggie**

### CODE REVIEW

File: Math/AddDigits.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Question Link: https://leetcode.com/problems/add-digits/
+ * Primary idea: Transform number into array of string and sum the values recursively until the result is achieved
+ *
+ * Time Complexity: O(1), Space Complexity: O(1)
+ *
+ */
+
+import UIKit
```
Comment (soapyigu) [inline Math/AddDigits.swift]: No need to import UIKit here

File: Math/AddDigits.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class Digits {
```
Comment (soapyigu) [inline Math/AddDigits.swift]: Could you please change the class name to AddDigits?

File: Math/AddDigits.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    func add (num: Int) -> Int {
+        let numString = String(num)
+        guard numString.characters.count > 1 else { return num }
```
Comment (soapyigu) [inline Math/AddDigits.swift]: Parentheses like "{}" should start with a new line.

**Comment by soapyigu** [comment]
Thank you for your contribution, could you please fix a tiny code style issue?

**Comment by Quaggie** [comment]
@soapyigu Of course I can (:

**Comment by Quaggie** [comment]
@soapyigu Fixed.

**Comment by soapyigu** [comment]
Awesome, thank you!

**Pull request merged**